docs/plugins/absubmit.rst

```diff
@@ -34,12 +44,20 @@ will also skip music that the analysis tool doesn't support.
 Configuration
 -------------
 
-To configure the plugin, make a ``absubmit:`` section in your configuration file. The available options are:
+To configure the plugin, make a ``absubmit:`` section in your configuration
+file. The available options are:
 
-- **auto**: Analyze every file on import. Otherwise, you need to use the ``beet absubmit`` command explicitly.
+- **auto**: Analyze every file on import. Otherwise, you need to use the
+  ``beet absubmit`` command explicitly.
   Default: ``no``
 - **extractor**: The absolute path to the `streaming_extractor_music`_ binary.
   Default: search for the program in your ``$PATH``
+- **force**: Analyze items and submit of AcousticBrainz data even for tracks
+  that already have it.
+  Default: ``no``.
+- **dry**: No analyze and submit of AcousticBrainz data but print out the
+  items which would be processed
```

Comment:
“No” -> “Do not”. Also, missing a period here.

Reply:
Okay thx for the fast review. Your project is fun to work with.